### ISSUE TITLE
[Feature request]: Search by Actor alias name

### ISSUE THREAD

**Issue by graphixillusion** [cooperdk/YAPO-e-plus#16]
As the title says, the search in the Actor tab, should work when using actor's alias name too and not just with the main name. Thank you

**Comment by cooperdk** [comment]
I agree this would be a nice feature. However, it would increase the loading time for the actor list by an immense amount, based on the way that YAPO was originally written. To make this work without issues, aliases would have to be incorporated into a new field in the actor table instead of being a reference to a field in another table. I'll give it some thought.
